src/web-ui/src/services/authorization.service.ts

```diff
@@ -0,0 +1,27 @@
+import {Injectable} from '@angular/core';
+import {APP_BASE_URL} from "../app/app.service";
+import {HttpClient} from "@angular/common/http";
+import {UserSignInRequest} from "../models/user.model";
+import {Router} from "@angular/router";
+import {ToastService} from "./toast.service";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthenticationService {
+
+  constructor(private http: HttpClient, private router: Router, private toastService: ToastService) {
+  }
+
+  public signinUser(request: UserSignInRequest) {
+    this.http.post<any>(`${APP_BASE_URL}/authenticate`, request).subscribe({
```

Comment:
Adres na który należy strzelić w celu uzyskania JWT to POST localhost:8086/api/v1/auth